Modules/AirCastingStyling/Sources/AirCastingStyling/CardShadow.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CardShadow.swift
+//  
+//
+//  Created by lunar  on 13/06/2022.
+//
+
+import SwiftUI
+
+public struct CardShadow: ViewModifier {
+    
+    public func body(content: Content) -> some View {
+        content
+            .shadow(color: .sessionCardShadow, radius: 3, x: 0, y: 2)
```

Comment:
I'm not sure why you decided to set x value to 0. If it was to make shadow symethrical, then you can actually add two `.shadow` modifiers, and this way add it on both sides. Like this:

```
.shadow(color: .sessionCardShadow, radius: 3, x: 1, y: 1)
.shadow(color: .sessionCardShadow, radius: 3, x: -1, y: -1)
```